simpledotorg/rtsl_utils
# #13779: Decouple docker from maven

Minimal changes to "Decouple docker from maven".

Tested locally and with the image added in dockerhub

steps to validate it works:

1. open docker_run/docker-compose.yaml
2. replace `0.0.0` by `ademarcq-13779.239` at line 19
3. run docker-compose up on in docker_run folder
4. check url http://localhost:8080/PrometheusDbExporter/metrics, you should have metrics
